tests/health.test.ts

```diff
@@ -0,0 +1,18 @@
+import request from 'supertest';
+import { Express } from 'express-serve-static-core';
+import { getServer, startMongoConnection } from "../app";
+
+let server: Express
+
+beforeAll(async () => {
+  startMongoConnection();
```

Comment:
Why do you need this for the health tests? Maybe if it fails otherwise, you should change your code design...

Reply:
it is, just changed name of file